jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -322,8 +322,18 @@ def step080_customize_java_args(script_dir, server_heap_settings, server_era) {
 
 def step081_customize_jruby_jar(script_dir, jruby_jar, server_era) {
     def jar_string = ""
+    def JRUBY_VERSION_SET = true
+
+    // This garbage can be replaced with either getBinding().hasVariable("JRUBY_VERSION")
+    // or via a params map (params.JRUBY_VERSION == null)
+    // https://wiki.jenkins.io/display/JENKINS/Pipeline+Groovy+Plugin#PipelineGroovyPlugin-2.18%28Sep23%2C2016%29
```

Comment:
So this garbage can go away when https://tickets.puppetlabs.com/browse/SERVER-2092 is done (it seems like updating the plugins are implicit in the description of that ticket)?

Reply:
Yes, exactly. Based on https://wiki.jenkins.io/display/JENKINS/Pipeline+Groovy+Plugin#PipelineGroovyPlugin-2.18%28Sep23%2C2016%29 we just need 2.7 or newer of the plugin.

Reply:
Would it be valuable to update this comment or comment in SERVER-2092 that we need to update this when we work that ticket?

Reply:
I'm happy to drop a note in SERVER-2092 about the sadness.